src/backend/dx11/src/lib.rs

```diff
@@ -268,11 +270,11 @@ impl hal::Instance for Instance {
                         heap_index: 0,
                     },
                     hal::MemoryType {
-                        properties: Properties::CPU_VISIBLE,
+                        properties: Properties::CPU_VISIBLE | Properties::CPU_CACHED,
```

Comment:
we expose no uncached CPU memory any more?

Reply:
Required for WR, not to mention that reads are cached already since we copy over from gpu memory to our "mapped" Vec<u8>